src/listeners/member-verification/modal-submit-validator.ts

```diff
@@ -225,6 +258,12 @@ export class InteractionCreateListener extends Listener<
       member.roles.add([guildSettings.verifiedRole, program.role], reason),
     ]);
 
+    if (locationRole !== '') {
```

Comment:
Rather than calling `member.roles.add` twice, we can combine this with the call above with something like:
```suggestion
    const roles = [guildSettings.verifiedRole, program.role];
    if (locationRole !== '') {
      roles.push(locationRole);
    }
    await Promise.all([
      member.setNickname(nickname, reason),
      member.roles.add(roles, reason),
    ]);
```